.github/workflows/deploy-docs.yaml

```diff
@@ -6,7 +6,7 @@ on:
       - main
       - tier4/proposal
     paths:
-      - "mkdocs.yaml"
+      - mkdocs.yaml
       - "**/*.md"
```

Comment:
[Question] What is the difference between a line that requires double quotation marks and one that does not?

Reply:
I remember it's whether starting from `*`.

Reply:
Thanks! I understood.